site/content/en/docs/drivers/hyperkit.md

```diff
@@ -29,6 +29,35 @@ If you are using `dnsmasq` and `minikube` fails, add `listen-address=192.168.64.
 
 If you are running other DNS servers, shut them off or specify an alternative bind address.
 
+### Hyperkit upgrade version
+The version of correct Hyperkit not being in the path could cause issues in starting minikube.
```

Comment:
I find this text difficult to read.

What do you think about:

```
### Upgrading HyperKit

New updates to macOS often require an updated hyperkit driver. To upgrade:

* If Docker for Desktop is installed, click on icon in your menu bar and select `Check for updates...`
* If you have Brew Package Manager, run: `brew upgrade hyperkit`
* As a final alternative, you may Install HyperKit from GitHub

To check your current version:

`hyperkit -v`

Reply:
ddone